client/js/_author/components/assessments/_edit_assessment.jsx

```diff
@@ -80,29 +77,18 @@ export class EditAssessment extends React.Component {
     );
   }
 
-  updateAssessment() {
+  updateAssessment(newFields) {
+    const { assessment } = this.props;
     this.props.updateAssessment(
       this.props.params.bankId,
```

Comment:
Would it make sense to merge javascript objects here instead of merging each property on the object manually? Then it might be more future proof if you ever need to add a property other than those 3.